__tests__/rpcProvider.test.ts

```diff
@@ -493,3 +493,49 @@ describeIfNotDevnet('waitForBlock', () => {
     expect(true).toBe(true); // answer without timeout Error (blocks have to be spaced with 16 minutes maximum : 200 retries * 5000ms)
   });
 });
+
+describeIfDevnet('EIP712 verification', () => {
+  // currently only in Devnet-rs, because can fail in Sepolia.
+  // to test in all cases once PR#989 implemented.
```

Comment:
This comment has been addressed through https://github.com/starknet-io/starknet.js/pull/1128

So, we can either remove it or add more tests

Reply:
If the problem is addressed, then it can be made in all networks.
Modified.

Reply:
I didn't verify about devnet vs testnet :D I just saw that PR#989 was merged

Reply:
Well, my PR#989 has not been merged : Tony closed it, implementing its own PR#1128 on this subject.
So, I suppose that he made the necessary tests to validate that it now works everywhere in all networks.